pkg/apis/core/v1alpha1/organizationroletemplate_types.go

```diff
@@ -72,6 +72,21 @@ type OrganizationRoleTemplateStatus struct {
 	// is a mechanism to map conditions to strings when printing the property.
 	// This is only for display purpose, for everything else use conditions.
 	Phase OrganizationRoleTemplatePhaseType `json:"phase,omitempty"`
+	// Targets holds different targets(Organization, Project) that this OrganizationRoleTemplate targets to.
+	Targets []OrganizationRoleTemplateTarget `json:"targets,omitempty"`
```

Comment:
Also, we should support the label selector for which Organization/Project this template should be applied to. Maybe create an issue about it and implement it in follow up PR? 

What do you think?

Reply:
Yeah, that makes sense, will create an issue about that

Reply:
great; link the issue here when it's created

Reply:
#38